### ISSUE TITLE
Slider still not working

### ISSUE THREAD

**Issue by livkx** [livkx/cadmus#2]
The slider from the last PR merged in, written by @mmarsalko seems to fix the no audio output issue, but there's still no visible slider for me just a blank button at the bottom.
![image](https://user-images.githubusercontent.com/15910716/98310416-c8550c00-1fc4-11eb-8842-48c70d0fbee2.png)

It's worth noting that we changed PyQT5 from 5.9.2 to 5.14.0 and sip from 4.19.8 to 5.4.0 - is anyone else experiencing the same with 0.0.3?

**Comment by mastastealth** [comment]
Same on Kubuntu: 
![image](https://user-images.githubusercontent.com/1221228/98371296-eb55df00-2009-11eb-94a7-0c8ca2893bf9.png)

**Comment by livkx** [comment]
Oops, I should probably have mentioned I’m also on Kubuntu 20.04. @mastastealth are you finding that the audio output is now functioning correctly though?

**Comment by mastastealth** [comment]
@livkx Yes, using my standard mic, and then the suppressed version **definitely** shows a difference.

**Comment by sfjuocekr** [comment]
Slider not working, clicking it also crashes cadmus :D
